### ISSUE TITLE
Add a config page to KCRM

### ISSUE THREAD

**Issue by paddyow** [Kademi/kademi-dev#1956]
Add a nice cog icon to configure KCRM 
- Setup and update funnel names
- Change drop down values for lost lead, close 
- Sources
- Lead types

This feature would only be available for users who have admin rights like sales managers. 

Thoughts?

**Comment by bradmac** [comment]
Yep. I think for smaller crm customers it seems counter intuitive to have an admin section seperate from the crm itself.
